README.md

```diff
@@ -145,17 +162,15 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
-
+"./cmake-build-debug/runMe" to run
```

Comment:
Well done. Not many folks got all 12 points for this.

My only observation: if you made `static const vector<entry> table{` a map, indexed by current state and next word,, you wouldn't have to search it.